FIAT/powell_sabin.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (C) 2024 Robert C. Kirby
+#
+# This file is part of FIAT (https://www.fenicsproject.org)
+#
+# SPDX-License-Identifier:    LGPL-3.0-or-later
+#
+# Written by Robert C. Kirby (robert.c.kirby@gmail.com), 2024
+
+from FIAT import dual_set, finite_element, macro, polynomial_set
+from FIAT.functional import (IntegralMomentOfNormalDerivative, PointDerivative,
+                             PointEvaluation)
+from FIAT.jacobi import eval_jacobi_batch
+from FIAT.quadrature_schemes import create_quadrature
+from FIAT.reference_element import TRIANGLE, ufc_simplex
+
+
+class QuadraticPowellSabin6DualSet(dual_set.DualSet):
```

Comment:
Do we need this class? can we not recycle the HermiteDualSet?

Reply:
No.  Hermite has an interior DOF for a total of 10, PS6 only has 9 dofs (jet at each vertex.)